ocs_ci/deployment/mce.py

```diff
@@ -0,0 +1,243 @@
+        """
+        Creates multiclusterengine operator
+
+        """
+        operatorgroup_yaml_file = templating.load_yaml(constants.MCE_OPERATOR_YAM)
+        operatorgroup_yaml = OCS(**operatorgroup_yaml_file)
+        try:
+            operatorgroup_yaml.create()
+            logger.info("mce OperatorGroup created successfully")
+        except exceptions.CommandFailed as ef:
+            if "multiclusterengine exists" in str(ef):
+                logger.info("multiclusterengine already exists")
+
+        cmd = "oc get mce multiclusterengine -o jsonpath='{.status}'"
+        cmd_res = exec_cmd(cmd, shell=True)
+        if cmd_res.returncode != 0:
+            logger.error(f"Failed to get multicluster engine status\n{cmd_res.stderr}")
+        else:
+            logger.info(
+                f"Multicluster engine version: {cmd_res.stdout.decode('utf-8')}"
+            )
+            assert (
+                cmd_res.stdout.decode("utf-8") == "Available"
+            ), "multiclusterengine is not is 'Available' status"
+
+    def create_mce_subscription(self):
+        """
+        Creates subscription for mce operator
+
+        """
+        mce_subscription_yaml_data = templating.load_yaml(
+            constants.MCE_SUBSCRIPTION_YAML
+        )
+
+        if config.DEPLOYMENT.get("mce_latest_stable"):
+            mce_subscription_yaml_data["spec"][
+                "source"
+            ] = constants.OPERATOR_CATALOG_SOURCE_NAME
+            mce_sub_channel = "stable-2.7"
+
+        mce_subscription_yaml_data["spec"]["channel"] = f"{mce_sub_channel}"
+        mce_subscription_manifest = tempfile.NamedTemporaryFile(
+            mode="w+", prefix="mce_subscription_manifest", delete=False
+        )
+        templating.dump_data_to_temp_yaml(
+            mce_subscription_yaml_data, mce_subscription_manifest.name
+        )
+        logger.info("Creating subscription for mce operator")
+        run_cmd(f"oc create -f {mce_subscription_manifest.name}")
+        OCP(
+            kind=constants.SUBSCRIPTION_COREOS,
+            namespace=self.namespace,
+            resource_name=constants.MCE_OPERATOR,
+        ).check_resource_existence(
+            should_exist=True, resource_name=constants.MCE_OPERATOR
+        )
+
+    def check_hypershift_namespace(self):
+        """
+        Check hypershift namespace created
+
+        """
+        logger.info(f"hypershift namespace {self.namespace} was created successfully")
+        is_hypershift_ns_available = self.ns_obj.is_exist(
+            resource_name=constants.HYPERSHIFT_NAMESPACE,
+        )
+        return is_hypershift_ns_available
+
+    def check_supported_versions(self):
+        """
+        Check supported ocp versions for hcp cluster creation
+
+        """
+        configmaps_obj = OCP(
+            kind=constants.CONFIGMAP,
+            namespace=constants.HYPERSHIFT_NAMESPACE,
+        )
+
+        assert configmaps_obj.is_exist(
```

Comment:
we have module with ocs-ci exceptions `ocs_ci.ocs.exceptions`, please use existing or add another with description instead of using assert